lua/harpoon/telescope.lua

```diff
@@ -0,0 +1,43 @@
+-- telescope.lua
+--
+-- A customizable finder for telescope.nvim
+-- (https://github.com/nvim-telescope/telescope.nvim)
+--
+-- Usage: :lua require("harpoon.telescope").marks({results_title = 'My Title'})
+local previewers = require('telescope.previewers')
+local pickers = require('telescope.pickers')
+local finders = require('telescope.finders')
+local sorters = require('telescope.sorters')
+local harpoon = require('harpoon')
+
+local default_options = {
+    results_title = 'Harpoon',
+    finder = finders.new_table(harpoon.get_mark_config().marks),
```

Comment:
Probably want to do this in the function.  Reason being that marks is not being updated

Reply:
That is a really good point. I didn't think about this when refactoring to allow overriding the options. I don't think it makes sense to allow overriding the finder anyway, now that I think of it.

Also, you said "you win". Were you avoiding adding a telescope integration since you have the quick menu? If that is the case, I have no problem just closing this.

Reply:
The issue should be resolved, but I'm totally fine if we don't merge this. Please don't merge it unless it is something you want in the project @ThePrimeagen